esp32_s3_korvo_2/idf_component.yml

```diff
@@ -7,6 +7,7 @@ targets:
 
 dependencies:
   idf: ">=5.0"
+  esp_lcd_ili9341: "^1"
   esp_lcd_touch_tt21100: "^1"
   esp_io_expander_tca9554: "^1"
```

Comment:
I'm having issues with `esp_io_expander`. Should it be listed here as 'public' dependency? Its header file is in in public `esp32_s3_korovo_2.h` header file.

See related job: https://github.com/espressif/esp-bsp/actions/runs/4067908226/jobs/7005819842

Reply:
cc @Lzw655

Reply:
Works OK now :)